src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/IncentiveSummaryService.kt

```diff
@@ -38,7 +38,7 @@ class IncentiveSummaryService(
     sortBy: SortColumn = SortColumn.NAME,
     sortDirection: Sort.Direction = Sort.Direction.ASC
   ): BehaviourSummary = coroutineScope {
-    val prisoners = prisonApiService.findPrisonersAtLocation(prisonId, locationId).toList()
+    val prisoners = prisonApiService.findPrisonersAtLocation(locationId).toList()
```

Comment:
`prisonId` was not used, [see other comment](https://github.com/ministryofjustice/hmpps-incentives-api/pull/236/files#r1010289259).